<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmUpdateSettings</class>
 <widget class="QDialog" name="FrmUpdateSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update settings</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>General:</string>
   </property>
   <widget class="QCheckBox" name="cbAutoUpdate">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>26</y>
      <width>341</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox { 
spacing: 5px;
}

QCheckBox::indicator {
width: 13px;
height: 13px;
}

QCheckBox::indicator:unchecked:disabled {
image: url(:/images/styles/checkbox_unchecked_dis.png);
}

QCheckBox::indicator:checked:disabled {
image: url(:/images/styles/checkbox_checked_dis.png);
}

QCheckBox::indicator:unchecked:enabled {
image: url(:/images/styles/checkbox_unchecked.png);
}

QCheckBox::indicator:checked:enabled {
image: url(:/images/styles/checkbox_checked.png);
}

QCheckBox::indicator:unchecked:enabled:hover {
image: url(:/images/styles/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:checked:enabled:hover {
image: url(:/images/styles/checkbox_checked_hover.png);
}
</string>
    </property>
    <property name="text">
     <string>Enable auto update</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>79</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Source:</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Url:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSource">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCangeSource">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>224</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Proxy:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>351</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="cbDirectConnection">
       <property name="styleSheet">
        <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
       </property>
       <property name="text">
        <string>Direct connection</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="cbUseProxy">
       <property name="styleSheet">
        <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
       </property>
       <property name="text">
        <string>Use HTTP proxy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtIp">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
         </property>
         <property name="placeholderText">
          <string>Please enter the proxy ip</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtpPort">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
         </property>
         <property name="text">
          <string>8080</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtProxyUser">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
         </property>
         <property name="placeholderText">
          <string>Please enter the username</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtProxyPassword">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Please enter a password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>391</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">:enabled {border: 1px solid black}
:disabled {border: 1px solid black}
:hover {background-color: rgb(195, 195, 195);}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="systray.qrc">
     <normaloff>:/images/close_dialog.png</normaloff>:/images/close_dialog.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdSave">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>391</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">:enabled {border: 1px solid black}
:disabled {border: 1px solid black}
:hover {background-color: rgb(195, 195, 195);}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="systray.qrc">
     <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>147</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Download source:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rbUseSecurepoint">
       <property name="styleSheet">
        <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
       </property>
       <property name="text">
        <string>Securepoint</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rbUseSourceForge">
       <property name="styleSheet">
        <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
       </property>
       <property name="text">
        <string>SourceForge</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="systray.qrc"/>
 </resources>
 <connections/>
</ui>
